content/commands/hugo.md

```diff
@@ -32,7 +32,7 @@ hugo [flags]
       --cacheDir string            filesystem path to cache directory. Defaults: $TMPDIR/hugo_cache/
       --canonifyURLs               if true, all relative URLs will be canonicalized using baseURL
       --cleanDestinationDir        remove files from destination not found in static directories
-      --config string              config file (default is path/config.yaml|json|toml)
+      --config string              config file(s), multiple files (a.toml,b.toml.c.toml) will be merged (default is path/config.yaml|json|toml)
```

Comment:
The content below /commands is autogenerated from code and this will be overwritten eventually.

Reply:
Might be better to have:

```markdown
### Options
<!-- Do NOT edit this section manually, it is auto-generated by ?? -->
```

*(where ?? be replaced with whatever is auto-generating this part.)*

Reply:
I see. 
If it's not about to be overwritten tomorrow, we could still add it and have you or me (depending on the complexity) modify the sentence directly to the hugo command --help function ?
I was about to look into adding the sentence to the cli --help anyway.

Reply:
https://github.com/gohugoio/hugo/blob/master/commands/hugo.go#L229